src/CareTogether.Core/Utilities/AppendBlobMultitenantEventLog.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Newtonsoft.Json;
+using OneOf;
+using OneOf.Types;
+
+
+namespace CareTogether.Utilities
+{
+    public class AppendBlobMultitenantEventLog<T>
+    {
+        private LogType _logType;
```

Comment:
Each of these private fields is written to only when the class is initialized in the constructor, so add a `readonly` modifier to them so they can't be changed later (e.g., `private readonly LogType _logType;`).